locales/de-de.json

```diff
@@ -0,0 +1,30 @@
+{
+  "about_page_title": "Über das Projekt",
+  "about_description": "JoinTheFediverse.net wurde von <a href=''>Stefan Bohacek</a> (<a href='mailto:stefan@stefanbohacek.com'>contact</a>) ins Leben gerufen. Übersetzungen wurden von folgenden wunderbaren Menschen beigesteuert:",
+  "about_cta_home": "Zurück zur Hauptseite",
+  "footer_about": "Über das Projekt",
+  "footer_view_source": "Quelltext anschauen",
+  "home_intro_what_is_fediverse": "Was ist das <mark>Fediverse</mark>?",
+  "home_intro_btn_explore": "Erkunden",
+  "home_intro_analogy": "Als Analogie zum Fediverse bietet sich <mark>E-Mail</mark> an.",
```

Comment:
There is a discussion happening, how we want to translate the art-word "Fediverse" in other languages: https://social.wake.st/@liaizon/108810406292194842 - It was suggested for the German language to use "Fediversum" and the English word "universe" is translated to "Universum".